js/newEddie.js

```diff
@@ -0,0 +1,15 @@
+var parents = document.getElementsByTagName("SELECT");
```

Comment:
this is un-necessary at this point and it doesn't look clean by any means. Please remove